README.md

```diff
@@ -111,39 +111,39 @@ Note:
 <!-- BEGIN ANSIBLE MANAGED BLOCK -->
 
 - Core
-  - [kubernetes](https://github.com/kubernetes/kubernetes) v1.32.2
-  - [etcd](https://github.com/etcd-io/etcd) v3.5.16
+  - [kubernetes](https://github.com/kubernetes/kubernetes) 1.32.2
+  - [etcd](https://github.com/etcd-io/etcd) 3.5.16
   - [docker](https://www.docker.com/) v26.1
   - [containerd](https://containerd.io/) v2.0.2
```

Comment:
Is there a special reason for retaining `v`? same with `scripts/readme_versions.md.j2` but I can't mention.